extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -603,30 +585,48 @@ export class AmpA4A extends AMP.BaseElement {
           if (!creativeParts) {
             return Promise.resolve();
           }
-          this.creativeSize_ = creativeParts.size || this.creativeSize_;
+          this.creativeSize_ = creativeParts.sizeInfo || this.creativeSize_;
           if (this.experimentalNonAmpCreativeRenderMethod_ !=
               XORIGIN_MODE.CLIENT_CACHE &&
               creativeParts.creative) {
             this.creativeBody_ = creativeParts.creative;
           }
-          if (!creativeParts.signature) {
+          if (!creativeParts.signatureInfo) {
             return Promise.resolve();
           }
           this.protectedEmitLifecycleEvent_('adResponseValidateStart');
-          return this.verifyCreativeSignature_(
-              creativeParts.creative, creativeParts.signature)
-              .then(creative => {
-                if (creative) {
-                  return creative;
+          const {signingServiceName, keypairId, signature} =
+              creativeParts.signatureInfo;
+          if (getMode().localDev) {
+            // localDev mode allows fake signature for the "fake" network.
+            if (signingServiceName == 'FAKESERVICE' &&
```

Comment:
At worst I would expect compiler to just remove that clause.  This is done throughout the codebase